BE/app/services/agents/copywriter.py

```
@@ -0,0 +1,302 @@
+                        "rationale": "비밀형 스타일로..."
+                    },
+                    ...
+                ]
+        
+        Raises:
+            ValueError: 키워드가 비어있거나 요약이 없는 경우
+            Exception: LLM API 호출 실패
+        """
+        # 입력 검증
+        if not keywords or len(keywords) == 0:
+            raise ValueError("키워드가 비어있습니다")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)